lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -448,6 +452,13 @@ type GetAssetDefault = "wallets"
     :> Capture "policyId" (ApiT TokenPolicyId)
     :> Get '[JSON] ApiAsset
 
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/mintBurnAssets
+type MintBurnAssets n = "wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "assets"
```

Comment:
Did you resolve the overlapping API path issue?
If not, don't fret too much about ugly URLs - because we will soon be rolling `PostMintBurnAssetDataT` into the transaction creation endpoint.

Reply:
Yep, resolved overlapping API path issue.